File: prime/src/main/java/org/ostelco/prime/storage/ProductDescriptionCache.java
package org.ostelco.prime.storage;


import org.ostelco.prime.storage.entities.Product;

public interface ProductDescriptionCache {
    void  addTopupProduct(String sku, long noOfBytes);

    boolean isValidSKU(String sku);

    Product getProductForSku(String sku);
}

File: prime/src/main/kotlin/org/ostelco/prime/storage/ProductDescriptionCache.kt
package org.ostelco.prime.storage


import org.ostelco.prime.model.Product

interface ProductDescriptionCache {

    fun addTopupProduct(sku: String, noOfBytes: Long)

    fun isValidSKU(sku: String): Boolean

    fun getProductForSku(sku: String): Product?
}
